### PR TITLE
[idea] quickfixes for implement abstract methods and constructors from super class

### PR DESCRIPTION
**Pull request by svenefftinge**

### CODE REVIEW

**Comment by svenefftinge** [comment]
Jenkins intellij-tests-master-latest

File: intellij/org.eclipse.xtext.idea/src/org/eclipse/xtext/idea/quickfix/AbstractIssueIntentionAction.xtend [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+abstract class AbstractIssueIntentionAction implements IntentionAction {
+	
+	@Accessors Issue issue
+	
+	override getFamilyName() {
+		getText()
+	}
+	
+	override isAvailable(Project project, Editor editor, PsiFile file) {
+		true
```
Comment (szarnekow) [inline intellij/org.eclipse.xtext.idea/src/org/eclipse/xtext/idea/quickfix/AbstractIssueIntentionAction.xtend]: Should be smth like PsiManager.isOnProject(..)

**Comment by svenefftinge** [comment]
Jenkins intellij-tests-master-latest

**Comment by szarnekow** [comment]
Test failure is unrelated and was already fixed on master in the meantime

**Pull request merged**